src/main/java/com/bulkbuy/enterprise/service/OrderService.java

```diff
@@ -21,7 +26,7 @@
     @Override
     public void Create(Order order) {
 
-        orders.add(order);
+        orderDAO.create(order);
```

Comment:
Replaced the local save to a List in the Service class to the one that I created within the DAO.